airflow/providers/tabular/sensors/tabular.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+from functools import cached_property
+from typing import TYPE_CHECKING
+
+from airflow.providers.tabular.hooks.tabular import TabularHook
+from airflow.sensors.base import BaseSensorOperator
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+PROPERTY_KEY_VTTS = "kafka.connect.vtts"
+
+
+class TabularVttsSensor(BaseSensorOperator):
```

Comment:
Would you mind adding params in the docstring? It would be nice for users to see them in the Python API documentation.

Reply:
I don't mind that at all, great suggestion. Added!